.ruby-version

```diff
@@ -1 +1 @@
-3.1.6
+3.0.6
```

Comment:
Was this important?

Reply:
It makes sense to have the Ruby version as the lowest supported minor version that way we're not using new features that we can't use yet since we're on Ruby 3.0